torch/_dynamo/__init__.py

```diff
@@ -53,6 +53,15 @@
     "list_backends",
 ]
 
+if torch.manual_seed is torch.random.manual_seed:
+    import torch.jit._builtins
+
+    # Wrap manual_seed with the disable decorator.
+    # Can't do it at its implementation due to dependency issues.
+    torch.manual_seed = disable(torch.manual_seed)
+    # Add the new manual_seed to the builtin registry.
+    torch.jit._builtins._register_builtin(torch.manual_seed, "aten::manual_seed")
```

Comment:
What is this one for?

Reply:
Mirrors the behavior of PyTorch legacy JIT for the original `torch.manual_seed`. Otherwise, it fails (IIRC) trying to trace through this function.